addons/calendar/models/calendar.py

```diff
@@ -709,6 +717,10 @@ def _compute_is_highlighted(self):
     privacy = fields.Selection([('public', 'Everyone'), ('private', 'Only me'), ('confidential', 'Only internal users')], 'Privacy', default='public', states={'done': [('readonly', True)]}, oldname="class")
     location = fields.Char('Location', states={'done': [('readonly', True)]}, track_visibility='onchange', help="Location of Event")
     show_as = fields.Selection([('free', 'Free'), ('busy', 'Busy')], 'Show Time as', states={'done': [('readonly', True)]}, default='busy')
+    activity_ids = fields.One2many('mail.activity', 'calendar_event_id', string='Activities', ondelete='cascade')
```

Comment:
There should not be any link to mail.activity in this commit. Purpose is to have clean and separated commits.